tests/text2graph/test_few_sentences.py

```diff
@@ -0,0 +1,19 @@
+import os
```

Comment:
We need an e2e workflow to run the test, with image build, service deploy, run test, stop service. Please refer to the other test scripts.